includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -635,71 +635,41 @@ public function get_order_items() {
 
 	/**
 	 * Get the tax rates/percentages for an item
-	 * @param  object $item order item
-	 * @param  object $order WC_Order
-	 * @param  bool $force_calculation force calculation of rates rather than retrieving from db
-	 * @return string $tax_rates imploded list of tax rates
+	 *
+	 * @param object $item order item
+	 * @param WC_Order|WC_Order_Refund $order WC_Order
+	 * @param bool $force_calculation force calculation of rates rather than retrieving from db
+	 *
+	 * @return string|float $tax_rates imploded list of tax rates
```

Comment:
I think it will be always `string`?

Reply:
Yes. I didn't understand the previous comment well and thought it was about this. Fixed.